packages/terra-clinical-onset-picker/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "terra-clinical-onset-picker",
+  "version": "0.1.0",
+  "description": "A complete form for selecting an onset date. Dates can be selected with a precision of day, month, or year.",
+  "main": "lib/OnsetPicker.js",
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rm -rf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "release:major": "npm test && node ../../scripts/release/release.js major",
+    "release:minor": "npm test && node ../../scripts/release/release.js minor",
+    "release:patch": "npm test && node ../../scripts/release/release.js patch",
+    "test": "npm run test:spec && npm run test:nightwatch-default",
```

Comment:
With the newest changes for update to terra-toolkit v2.0.0, we have updated these tests scripts. [Here](https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-action-header/package.json#L54) is an example of the new changes

Reply:
Updated scripts for terra-toolkit 2.0.0